subprojects/ide/src/main/java/org/gradle/plugins/ide/internal/tooling/GradleProjectBuilder.java

```diff
@@ -65,7 +66,14 @@ private DefaultGradleProject buildHierarchy(Project project) {
                 .setChildren(children);
 
         gradleProject.getBuildScript().setSourceFile(project.getBuildFile());
-        gradleProject.setTasks(tasks(gradleProject, (TaskContainerInternal) project.getTasks()));
+
+        // Internal system property to investigate model loading performance in IDEA/Android Studio
+        String projectOptions = System.getProperty("org.gradle.unsafe.GradleProjectBuilderOptions", "unmodified");
```

Comment:
Let's do a couple of things here...

1. Rename this to `org.gradle.internal.` instead of unsafe as we'll remove it eventually. 
2. List the different values in a comment here and what they would do/mean.

From what I can tell, we have
- no value/`unmodified` which means "do what we do today"
- `skip_task_graph_realization` which means create artificial tasks from just the task names and send them over
- `skip_task_serialization` which means to realize all tasks, but don't send them over. 

`unmodified` - `skip_task_graph_realization` = overhead to realize tasks
`unmodified` - `skip_task_serialization` = overhead to serialize tasks

This looks like what we're looking for.

Reply:
Property renamed and documented.